src/main/java/io/trino/plugin/tiledb/TileDBRecordCursor.java

```diff
@@ -542,12 +541,13 @@ else if (attBounds.getSecond() == null) {
 
     /**
      * Returns the Query condition for the given bound.
+     *
      * @param attr The attribute
      * @param isString True if the attribute is String
      * @param bound The bound
      * @param op TIleDB operator
```

Comment:
typo

Reply:
Same reason as above, parameters in java docs now need to have one empty line preceding them